docs/kyma/05-01-overview.md

```diff
@@ -3,53 +3,71 @@ title: Overview
 type: Configuration
 ---
 
```

Comment:
@Tomasz-Smelcerz-SAP you've mentioned deletion of cluster config yaml but I cannot see it being deleted here?

Reply:
Removed.